### ISSUE TITLE
Adding a link to your reading list video

### ISSUE THREAD

**Issue by dandange8005** [dandange8005/Ultra-Essentials#49]
[Reading list](https://xerte.cardiff.ac.uk/play_18321#buildingCoursesection9) > Adding a link to your reading list or reading list item

It is reported that: Adding a link to your reading list video has voiceover but the screen is not visible after the first couple of minutes.

<img width="935" alt="image" src="https://github.com/dandange8005/Ultra-Essentials/assets/5180820/93ff628c-9e96-4dad-bd13-77b24f86719f">

**Comment by dandange8005** [comment]
I have checked and found no issue with the video's visibility.
